Demo/Demo/ViewModels/CardPaymentViewModel.swift

```diff
@@ -10,15 +10,34 @@ class CardPaymentViewModel: ObservableObject, CardDelegate {
 
     private var cardClient: CardClient?
 
-    func createOrder(amount: String, selectedMerchantIntegration: MerchantIntegration, intent: String) async throws {
+    func createOrder(
+        amount: String,
+        selectedMerchantIntegration: MerchantIntegration,
+        intent: String,
+        shouldVault: Bool,
+        customerID: String? = nil
+    ) async throws {
         // might need to pass in payee as payee object or as auth header
 
         let amountRequest = Amount(currencyCode: "USD", value: amount)
         // TODO: might need to pass in payee as payee object or as auth header
```

Comment:
Are these TODOs still relevant? Also they seem to be duplicate, we can remove 1

Reply:
Great catch! I left a comment I made for myself not realizing I already added one a while back.
I think it's good to note for now as reminder for different scenarios we may address in the demo app.